elastic/kibana
# [Synthetics] add zip url deprecation message to package and Uptime overview

## Summary

Resolves https://github.com/elastic/kibana/issues/140611

Adds a deprecation notice for zip url monitor source

Within the Synthetics Integration package:
![image](https://user-images.githubusercontent.com/11356435/192010274-5d173596-84c9-4cdb-a392-84475d721b9e.png)

On the Uptime overview page
![image](https://user-images.githubusercontent.com/11356435/192347607-bac5a87c-3806-498d-8e0f-553da9afd0e4.png)


### Testing
1. Navigate to `app/integrations/detail/synthetics-0.10.2/overview`.
2. Select browser monitor
3. Confirm that inline script is the first tab
4. Create an integration for a browser monitor WITHOUT zip url, (either recorder or inline).
5. Navigate to Uptime overview page. Verify that a deprecation notice DOES NOT appear
6. Next create an integration for a browser monitor WITH a zip url. There should be a deprecation notice displayed while creating the integration
7. Navigate to the Uptime overview page. Verify that the deprecation notice DOES appear
8. Dismiss the notification. Ensure it disappears
9. Refresh the page, ensure the notification still does not appear
10. Close your browser and start a new session. Ensure the notification DOES appear
11. Delete the zip url integration
12. Ensure that notice on the overview page DOES NOT APPEAR

### Checklist

Delete any items that are not applicable to this PR.
